otter/cloud_client.py

```diff
@@ -220,6 +302,21 @@ def scoped_performer(dispatcher, service_request):
     perform(new_disp, tenant_scope.effect.on(box.succeed, box.fail))
 
 
+def get_cloud_client_dispatcher(reactor, authenticator, log, service_configs):
+    """
+    Get a dispatcher suitable for running :obj:`ServiceRequest` and
+    :obj:`TenantScope` intents.
+    """
+    # this throttler could be parameterized but for now it's basically a hack
+    # that we want to keep private to this module
```

Comment:
Do you want to create a tech-debt ticket that describes the ideal throttler? Or an issue in effect?